homeassistant/components/ezviz/alarm_control_panel.py

```diff
@@ -0,0 +1,113 @@
+"""Support for Ezviz alarm."""
+from __future__ import annotations
+
+from typing import Any
+
+from pyezviz.constants import DefenseModeType
+from pyezviz.exceptions import HTTPError
+
+from homeassistant.components.alarm_control_panel import AlarmControlPanelEntity
+from homeassistant.components.alarm_control_panel.const import (
+    SUPPORT_ALARM_ARM_AWAY,
+    SUPPORT_ALARM_ARM_NIGHT,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.restore_state import RestoreEntity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    ATTR_AWAY,
+    ATTR_HOME,
+    ATTR_SLEEP,
+    DATA_COORDINATOR,
+    DOMAIN,
+    MANUFACTURER,
+)
+from .coordinator import EzvizDataUpdateCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Ezviz alarm control panel."""
+    coordinator: EzvizDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id][
+        DATA_COORDINATOR
+    ]
+
+    async_add_entities([EzvizAlarm(coordinator)])
+
+
+class EzvizAlarm(CoordinatorEntity, AlarmControlPanelEntity, RestoreEntity):
+    """Representation of a Ezviz alarm control panel."""
+
+    coordinator: EzvizDataUpdateCoordinator
+    _attr_name = "Ezviz Alarm"
+    _attr_supported_features = SUPPORT_ALARM_ARM_AWAY | SUPPORT_ALARM_ARM_NIGHT
+    _attr_code_arm_required = False
+
+    def __init__(self, coordinator: EzvizDataUpdateCoordinator) -> None:
+        """Initialize the sensor."""
+        super().__init__(coordinator)
+        self._attr_state = STATE_ALARM_DISARMED
+        self._model = "Ezviz Alarm"
+        self._attr_unique_id = "Ezviz Alarm"
```

Comment:
This is not allowed as a unique ID
https://developers.home-assistant.io/docs/entity_registry_index?_highlight=unique#unique-id-requirements

Reply:
Hi @ludeeus,

Can I remove self._attr_unique_id?